{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/static_source/css/base.scss

```diff
@@ -27,8 +23,8 @@
  *
  */
 
-@import "@/../templates/components/links/links.scss";
-@import "@/../templates/components/messages.scss";
+
+@import "components/components";
```

Comment:
do we need this anymore?

Reply:
dependent on whether we reorganize messages in the other PR or not, will wait to resolve that and then update this accordingly